ui/src/app/cohort-search/search-group/search-group.component.tsx

```diff
@@ -819,21 +820,33 @@ export const SearchGroup = withCurrentWorkspace()(
                   </div>
                 ))}
                 {/* Criteria menu for temporal group 1 items */}
-                <div style={styles.cardBlock}>
-                  <Menu
-                    style={styles.menu}
-                    appendTo={document.body}
-                    model={this.criteriaMenuItems(1)}
-                    popup
-                    ref={(el) => (this.temporalCriteriaMenu = el)}
+                {/* if feature flag is enabled - render the new component CohortCriteriaMenu */}
+                {serverConfigStore.get().config.enableUniversalSearch ? (
+                  <CohortCriteriaMenu
+                    launchSearch={(criteria, temporalGroup, searchTerms) =>
+                      this.launchSearch(criteria, temporalGroup, searchTerms)
+                    }
+                    menuOptions={criteriaMenuOptions}
+                    temporalGroup={0}
```

Comment:
`temporalGroup` should be 1 here since we want these selections in the bottom group. This should fix the issue @freemabd pointed out.

Reply:
done